test/e2e/kubectl/kubectl.go

```diff
@@ -1939,6 +1939,43 @@ metadata:
 		})
 	})
 
+	ginkgo.Describe("Kubectl events", func() {
+		var podName string
+
+		ginkgo.BeforeEach(func() {
+			podName = "e2e-test-httpd-pod"
+		})
+
+		ginkgo.AfterEach(func() {
+			framework.RunKubectlOrDie(ns, "delete", "pods", podName)
+		})
+
+		framework.ConformanceIt("should show event when pod is created ", func() {
```

Comment:
You can't go directly to conformance, there are some requirements of stability, first starts as a normal e2e

Reply:
Thanks a lot @aojea for review. Do you have any handy tests as an example(to understand what normal e2e looks like)?, I could not find `framework.e2e` in codebase.

Reply:
I think, I understand what you mean, just running with `ginkgo.It`